scripts/generator/graphql_generator/csharp/SDK/Serializable.cs.erb

```diff
@@ -1,14 +1,9 @@
-#if !SHOPIFY_MONO_UNIT_TEST
 namespace <%= namespace %>.SDK {
     using System;
     using System.Collections;
     using System.Collections.Generic;
-    using UnityEngine;
 
-    public class Serializable {
-        public string ToJsonString() {
-            return JsonUtility.ToJson(this);
-        }
+    public interface Serializable {
+        string ToJsonString();
```

Comment:
Since all implementations of `Serializable` simply do:
```
public string ToJsonString() {
    return Json.Serialize(ToJson());
}
```
We should just make `Serializable` be abstract and add the above here.